xml/System.IO.Compression/ZipFileExtensions.xml

```diff
@@ -121,7 +121,7 @@
         <exception cref="T:System.IO.PathTooLongException">In <paramref name="sourceFileName" />, the specified path, file name, or both exceed the system-defined maximum length. For example, on Windows-based platforms, paths must not exceed 248 characters, and file names must not exceed 260 characters.</exception>
         <exception cref="T:System.IO.DirectoryNotFoundException">
           <paramref name="sourceFileName" /> is invalid (for example, it is on an unmapped drive).</exception>
-        <exception cref="T:System.IO.IOException">The file specified by <paramref name="sourceFileName" /> cannot be opened.</exception>
+        <exception cref="T:System.IO.IOException">The file specified by <paramref name="sourceFileName" /> cannot be opened, or is too large to be updated (current limit is cref="F:System.Int32.MaxValue">).</exception>
```

Comment:
@BillWagner please bring back the opening bracket in front of the cref link. Now it is
>(current limit **is cref**="F:System.Int32.MaxValue">)